decidim-debates/lib/decidim/debates/engine.rb

```diff
@@ -97,6 +97,37 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_debates.register_events" do
+        config.to_prepare do
+          Decidim::EventsManager.subscribe("decidim.debates.create_debate:after") do |_event_name, data|
+            Decidim::EventsManager.publish(
+              event: "decidim.events.debates.debate_created",
+              event_class: Decidim::Debates::CreateDebateEvent,
+              resource: data[:resource],
+              followers: data[:resource].author.followers,
+              extra: {
+                type: "user"
+              }
+            )
```

Comment:
Looking at this made me think this is kind of redundant but then I realized what the `EventsManager` is supposed to represent.

As I understand it, the `EventsManager` class is supposed to deal with the Decidim events that cause in-app or email/push notifications. Now we are using the same class to also subscribe to the command notifications which makes this super confusing to understand and also makes the "EventsManager" class ambiguous in many senses.

So my suggestions to solve this:
1. Do not use `EventsManager` to subscribe to any other events but `decidim.events.*`
2. Do not use `EventsManager` to publish any other events but `decidim.events.*`
3. Preserve the `EventsManager` to solely purposes related to the application notifications (i.e. what the `EventPublisherJob` is dealing with)

Reply:
The issue should be fixed by [cf9a7a0](https://github.com/decidim/decidim/pull/11117/commits/cf9a7a04d10447eb1344eb63d60f076ff4e9a734)